test/v200/utils/api/test_utils.go

```diff
@@ -0,0 +1,173 @@
+package api
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"strconv"
+	"strings"
+	"testing"
+
+	common "github.com/devfile/api/v2/test/v200/utils/common"
+	"github.com/santhosh-tekuri/jsonschema"
+	"sigs.k8s.io/yaml"
+)
+
+const (
+	// numDevfiles : the number of devfiles to create for each test
+	numDevfiles = 5
+
+	schemaFileName = "../../../schemas/latest/ide-targeted/devfile.json"
+)
+
+var schemas = make(map[string]SchemaFile)
+
+type SchemaFile struct {
```

Comment:
desc would be nice

Reply:
OK, next PR

Reply:
Added comment in new commit - may remove later.